src/test/fuzz/deserialize.cpp

```diff
@@ -23,6 +24,14 @@
 
 #include <test/fuzz/fuzz.h>
 
+void initialize()
+{
+#if BLOCKUNDO_DESERIALIZE || COINS_DESERIALIZE || TXOUTCOMPRESSOR_DESERIALIZE || TXUNDO_DESERIALIZE
+    // Fuzzers using pubkey must hold an ECCVerifyHandle.
+    static const auto verify_handle = MakeUnique<ECCVerifyHandle>();
+#endif
```

Comment:
Might just initialize it for all of them. The overhead is zero.

Reply:
Done! Please re-review :)